trace/Sampling.md

```diff
@@ -0,0 +1,35 @@
+# Sampling
+
+This document is about the sampling bit, sampling decision, samplers and how and when does 
+OpenCensus sample traces. A sampled traces is a traces that gets exported via the configured
+exporters.
+
+## Sampling Bit (propagated via TraceOptions)
+
+Sampling bit is always set only at the start of a Span, and is using a `Sampler`
```

Comment:
I think we need an overview of propagation (vs a state chart). or at least a separate section on how to achieve B3 style "trust the caller" decisions. The fact that a sampler is required for each span may seem complex to those who currently only do it at the head of a trace. having a default as create or trust. would match zipkin and zipkin-like systems.. worth either making default do that or making it easy to have the default do that imho. I don't think probability 0 is intuitive as for ex in zipkin this is same as Never

Reply:
so the default is Probability(1.0/10000):
* As Probability sampler it keeps the sampling probability;
* It inflates the sampling rate to 1/10000 (in case no-sampled);